MeshKernelNET/src/MeshKernelNETCore/Native/SplinesToCurvilinearParametersNative.cs

```diff
@@ -13,14 +13,14 @@ public struct SplinesToCurvilinearParametersNative
         public double AspectRatioGrowFactor { get; set; }
 
         /// <summary>
-        /// * Average mesh width on center spline (0.005)
+        /// * Average mesh width on center spline (500.0)
```

Comment:
My memory is a bug numb on this. I know we had a discussion about it, but this was the new default?

Reply:
Original value of 500 was restored. This is coherent with the default in [interactor.](https://github.com/Deltares/MeshKernelNET/pull/11#discussion_r1245288592)